.travis.yml

```diff
@@ -38,6 +38,15 @@ install:
   - npm install -g npm@6
   - node --version
   - npm --version
+  - curl -sSf https://sh.rustup.rs | sh -s -- -y --verbose
+  - PATH=$PATH:$HOME/.cargo/bin
+  - rustup install nightly-2018-06-23
```

Comment:
Late to this, sorry, but wouldn't it make sense to use the same version we specify for CI and docker? (currently 2018-07-14)

Reply:
Yeap good idea!